src/tree.c

```diff
@@ -55,7 +55,7 @@ static struct ofi_rbnode *ofi_rbnode_alloc(struct ofi_rbmap *map)
 {
 	struct ofi_rbnode *node;
 
-	if (!map->free_list)
+	if (OFI_UNLIKELY(!map->free_list))
```

Comment:
Please remove the UNLIKELY.  In the vast majority of cases, developers get this wrong.  It may be common that the fee list is empty.

Reply:
@shefty Done.